app/controllers/api/v1/games_controller.rb

```diff
@@ -9,4 +11,14 @@ def index
       render json: Game.all
     end
   end
+
+  def create
+    game = Game.new(name: params[:name], description: params[:description], min_player_count: params[:min_player_count], max_player_count: params[:max_player_count])
```

Comment:
ideally, you would say something like:

```ruby
game = Game.new(game_params)

# and then have a strong params method
def game_params
  #...
end
```

Reply:
should this method be in the api controller and private?